### PR TITLE
Advice for forkers

### PR DESCRIPTION
**Pull request by crush-157**
Just added some advice to forkers explaining about cloning from the fnproject/cli repo and adding their fork as an additional remote, rather than cloning from their own and adding the original as an upstream remote.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -12,6 +12,16 @@ We welcome all contributions
...
 6. Good Job! Thanks for being awesome!
 
+## Advice for Forkers ##
+
+We do not advise cloning from your own fork (see [Getting the Repository](#Getting_the_Repository) below).
+
+Instead we suggest:
+1. Clone from the original repo
+2. Add your fork as a remote
+3. Push your changes your fork
```
Comment (treeder) [inline CONTRIBUTING.md]: typo: "to your fork"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -30,7 +40,7 @@ asking for suggestions on how to address the documentation part.
 - [Go](https://golang.org/doc/install)
 - [Dep](https://github.com/golang/dep)
 
-### Getting the Repository ###
+### Getting the Repository <a name="Getting_the_Repository"></a> ###
```
Comment (treeder) [inline CONTRIBUTING.md]: GitHub already adds anchors to all the titles, so this is unnecessary.

File: CONTRIBUTING.md (hunk 1 above)
Comment (crush-157) [reply CONTRIBUTING.md]: Thanks.

File: CONTRIBUTING.md (hunk 2 above)
Comment (crush-157) [reply CONTRIBUTING.md]: Noted.  Thanks

**Comment by crush-157** [comment]
Changes made

**Pull request merged**